docs/inference-providers/providers/baseten.md

```diff
@@ -0,0 +1,42 @@
+<!---
+WARNING
+
+This markdown file has been generated from a script. Please do not edit it directly.
+
+### Template
+
+If you want to update the content related to baseten's description, please edit the template file under `https://github.com/huggingface/hub-docs/tree/main/scripts/inference-providers/templates/providers/baseten.handlebars`.
+
+### Logos
+
+If you want to update baseten's logo, upload a file by opening a PR on https://huggingface.co/datasets/huggingface/documentation-images/tree/main/inference-providers/logos. Ping @wauplin and @celinah on the PR to let them know you uploaded a new logo.
+Logos must be in .png format and be named `baseten-light.png` and `baseten-dark.png`. Visit https://huggingface.co/settings/theme to switch between light and dark mode and check that the logos are displayed correctly.
+
+### Generation script
+
+For more details, check out the `generate.ts` script: https://github.com/huggingface/hub-docs/blob/main/scripts/inference-providers/scripts/generate.ts.
+--->
+
+# Cerebras
+
+> [!TIP]
+> All supported Baseten models can be found [here](https://huggingface.co/models?inference_provider=baseten&sort=trending)
+
+<div class="flex justify-center">
```

Comment:
As mentioned in the [Hub PR](https://huggingface.co/datasets/huggingface/documentation-images/discussions/559), the light and dark logos are to be swapped: https://huggingface.co/datasets/huggingface/documentation-images/discussions/559 (to be done in the Hub PR, not here)